telegram/ext/_extbot.py

```diff
@@ -4216,6 +4218,46 @@ async def get_star_transactions(
             api_kwargs=self._merge_api_rl_kwargs(api_kwargs, rate_limit_args),
         )
 
+    async def send_paid_media(
+        self,
+        chat_id: Union[str, int],
+        star_count: int,
+        media: Sequence[Union["InputPaidMediaPhoto", "InputPaidMediaVideo"]],
+        caption: Optional[str] = None,
+        parse_mode: ODVInput[str] = DEFAULT_NONE,
+        caption_entities: Optional[Sequence["MessageEntity"]] = None,
+        show_caption_above_media: Optional[bool] = None,
+        disable_notification: ODVInput[bool] = DEFAULT_NONE,
+        protect_content: ODVInput[bool] = DEFAULT_NONE,
+        reply_parameters: Optional["ReplyParameters"] = None,
+        reply_markup: Optional[ReplyMarkup] = None,
```

Comment:
uh, we need logic for arbitrary context data here …

Reply:
`Bot.send_paid_media` calls `self._send_message`, which is overriden by `ExtBot` to insert callback data, so we don't have to do anything?

Reply:
ah, righto. Still wondering if we should add a new test case. Haven't worked on those tests for a long time :/

Reply:
Apparently we also only test with `send_message` for the other ones. I'll resolve